lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEFYarn.cs

```diff
@@ -95,10 +96,15 @@ private void Run()
                 .Build();
 
             var result = _reefClient.SubmitAndGetJobStatus(helloJobRequest);
-
             LogApplicationReport();
+
+            //// This is to test Kill Job Application
+            //// KillJobApplication(result.AppId);
+
+            LogApplicationReport(result.AppId);
```

Comment:
It was to check the app status after killing it. Also to test LogApplicationReport with appId. previous call doesn't have appId. As KillJobApplication() is commented. I can remove this call.